crypto/ui/ui_openssl.c

```diff
@@ -443,6 +443,14 @@ static int open_console(UI *ui)
         if (errno == ENODEV)
             is_a_tty = 0;
         else
+#  endif
+#  ifdef ENOSYS
+            /*
+             * z/OS Mainframe can reurn E_NO_SYS (The function is not implemented)
+             */
```

Comment:
Please indent the comment to be lined up with the statement below. Again this has nothing to do with EBCDIC and everything to do with z/os so should probably be in a different PR.

Reply:
Removed